Directory.Build.props

```diff
@@ -251,6 +251,10 @@
   </PropertyGroup>
 
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+  
+  <PropertyGroup>
+    <Copyright>© .NET Foundation and Contributors</Copyright>
+  </PropertyGroup>
```

Comment:
Do we need PackageLicenseExpression as well then?

Reply:
I have a new PR: https://github.com/dotnet/corefx/pull/35524 will close this one